_sass/_uswds-theme-custom-styles.scss

```
@@ -108,3 +108,10 @@ a {
 footer {
   order: unset;
 }
+
+// increase height for hero section
+@media (min-width: 640px) {
+  .hero__container {
+      max-height: 550px;
+  }
+}
```

It's not important but the convention is to use a newline at the end of a file

---

What do you mean by "using a newline at the end of a file?" What would that look like here?

---

```suggestion
}

```
Just pressing return / enter at the end of the file, like this